inc/wp-components/classes/class-component.php

```diff
@@ -262,11 +341,18 @@ public function callback( $callable ) {
 	/**
 	 * Execute a function on each child of this component.
 	 *
-	 * @param callable $callback Callback function.
+	 * @param string        $group Component group on which to call the callback.
+	 * @param callable|bool $callback Callback function.
 	 * @return self
 	 */
-	public function children_callback( $callback ) : self {
-		$this->children = array_map( $callback, $this->children );
+	public function children_callback( $group, $callback = false ) : self {
+		// If valid group, map over the group, otherwise map over children.
+		if ( $this->is_valid_group( $group ) ) {
+			$this->component_groups[ $group ] = array_map( $callback, $this->component_groups[ $group ] );
```

Comment:
As far as I know, the only valid default for a callable is null. Worth checking if L351 throws warnings or errors when `$callback` is not passed.